### PR TITLE
[WIP] Test Torch installation and model zoo

### PR DESCRIPTION
**Pull request by bryanyzhu**

### CODE REVIEW

**Comment by mli** [comment]
Job PR-1489-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1489/1/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1489-2 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1489/2/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1489-3 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1489/3/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1489-8 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1489/8/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1489-9 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1489/9/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1489-10 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1489/10/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1489-11 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1489/11/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1489-12 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1489/12/index.html 
Code coverage of this PR:  vs. Master:

File: docs/conf.py [hunk 1]
```diff
@@ -105,6 +105,10 @@
     'backreferences_dir': None,
 }
 
+# Disable tutorial if needed
+if os.environ.get('GLUONCV_DISABLE_TUTORIALS') == 'True':
```
Comment (zhreshold) [inline docs/conf.py:109]: os.environ.get('GLUONCV_DISABLE_TUTORIALS', 'False') == 'True'

**Comment by mli** [comment]
Job PR-1489-14 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1489/14/index.html 
Code coverage of this PR:  vs. Master:

**Review by zhreshold — APPROVED** [review]
lgtm!

**Pull request merged**